docs/releasenotes.asciidoc

```diff
@@ -4,20 +4,50 @@
 The {version} release of Kibana requires Elasticsearch {esversion} or later.
 
 [float]
+[[breaking]]
+== Breaking Change
+
+* {k4issue}5591[Issue 5591]: The command-line plugin tool no longer supports Github.
+
+[float]
 [[enhancements]]
 == Enhancements
 
-// * {k4issue}5109[Issue 5109]: Adds custom JSON and filter alias naming for filters.
-// * {k4issue}1726[Issue 1726]: Adds a color field formatter for value ranges in numeric fields.
-// * {k4issue}4342[Issue 4342]: Increased performance for wildcard indices.
-// * {k4issue}1600[Issue 1600]: Support for global time zones.
-// * {k4pull}5275[Pull Request 5275]: Highlighting values in Discover can now be disabled.
-// * {k4issue}5212[Issue 5212]: Adds support for multiple certificate authorities.
-// * {k4issue}2716[Issue 2716]: The open/closed position of the spy panel now persists across UI state changes.
+* {k4issue}6387[Issue 6387]: A new look for Kibana: new logo and UI, improved layout and navigation.
+* {k4pull}6402[Pull Request 6402]: Refactored the command-line plugin management tool to support cross-stack plugin 
```

Comment:
For release notes, how about simply "New command-line plugin..." rather than "Refactored the..."